RedHat-UX/red-hat-design-system
# docs: example image css

## What I did

1. Resolves [#979](https://github.com/RedHat-UX/red-hat-design-system/issues/979). Adjusted default example image size since the example images are 2X the desired size by using: `width: auto; transform: scale(0.5); margin: -7.5% auto;`


## Testing Instructions

1. Check [DP](https://deploy-preview-991--red-hat-design-system.netlify.app/) for issues with image size.

## Notes to Reviewers
